src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
make this variable global and use it here

Reply:
fixed